isis/src/base/apps/bandnorm/bandnorm.h

```diff
@@ -0,0 +1,11 @@
+#ifndef std2isis_h 
+#define std2isis_h
+
+#include "UserInterface.h"
+
+namespace Isis {
+  extern void bandnorm(Cube *icube, UserInterface &ui);
```

Comment:
Surprised this doesn't need the Cube include

Reply:
Ye, I kinda messed with that, I imagine everyone that includes it has Cube.h included? In which case I should add it here.